packages/go/tts/router_test.go

```
@@ -0,0 +1,666 @@
+
+	st, err := r.Synthesize(Request{Text: "今天讲个小熊的故事", Tier: 0, TurnID: "t1"})
+	if err != nil {
+		t.Fatalf("Synthesize: %v", err)
+	}
+	var got [][]byte
+	for i := 0; i < 2; i++ { // 已播半句（2 chunks）
+		c, err := st.Next()
+		if err != nil {
+			t.Fatalf("前 %d chunk 须正常：got %v", i+1, err)
+		}
+		got = append(got, c.Data)
```

golangci-lint (2.12.2)

[error] SA4010: this result of append is never used, except maybe in other appends

(staticcheck)